addons/website_sale_stock/static/tests/tours/website_sale_stock_product_configurator.js

```diff
@@ -23,12 +23,20 @@ registry
             // Assert that it's impossible to add less than 1 product (only for the main product).
             configuratorTourUtils.setProductQuantity("Main product", 0),
             configuratorTourUtils.assertProductQuantity("Main product", 1),
-            configuratorTourUtils.decreaseProductQuantity("Main product"),
```

Comment:
Could you also remove the `decreaseProductQuantity` function itself? I believe it was only used here.